lib/Slic3r/Print/SupportMaterial.pm

```diff
@@ -22,10 +22,6 @@ use constant DEBUG_CONTACT_ONLY => 0;
 # increment used to reach MARGIN in steps to avoid trespassing thin objects
 use constant MARGIN_STEP => MARGIN/3;
 
-# generate a tree-like structure to save material
-use constant PILLAR_SIZE    => 2.5;
```

Comment:
https://travis-ci.org/slic3r/Slic3r/jobs/377987179

These are being referenced in a few tests. Please update the tests to pull the default values of the config instead. (in the tests the config is usually initialized as $config to defaults and you can reference it as $config->pillar_size in those tests or $config->pillar_spacing)

Reply:
On the other hand, it built so that's good :)

If you drop a copy of local-lib from a build package into your Slic3r source directory, you can use the following to just build XS and usually avoid having to do a full "perl Build.pl"

On Windows, open up Powershell and cd to the build source.
`Invoke-Expression "$(perl -Mlocal::lib=./local-lib)"` will configure that window appropriately, so you can go into the xs directory and run `./Build install` and it'll just compile Slic3r without going to get any of the Perl dependencies.

Reply:
I am using macOS, but I managed to run those commands. How do I actually launch Slic3r after that though? Either way, I think I fixed the tests in 1bad4af.

Reply:
On OSX, the magic is `eval $(...)`

You run Slic3r as normal (`perl slic3r.pl`), and you can run the tests as Perl scripts or use the prove command . It lasts until you close that terminal.

Reply:
On OSX I highly recommend using perl brew if you aren't.

Reply:
I don't really understand your instructions. `local-lib` from where? What commands do I run to get the GUI working, starting with checking out my branch?

I think the issue is that I can't figure out how to install wxWidgets. I have tried many different ways but none of them have succeeded (except installing through Homebrew, but then `Alien::wxWidgets` wouldn't recognize it). Could you please test for me that the UI works properly? I fixed the tests.